qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -31,13 +31,22 @@
 
 
 class InstructionToSignals:
-    """Converts pulse instructions to Signals for the Aer simulator."""
+    """Converts pulse instructions to Signals to be used in models.
+
+    Qiskit-dynamics simulates the time evolution using the :class:`Signal` class.
```

Comment:
I think this general blurb about dynamics using `Signal`s, so simulating pulse schedules requires a means of conversion should go into the module doc string.

Reply:
See here: 094e9f2